src/mailmojo-plugin.php

```diff
@@ -50,6 +53,19 @@ public static function getInstance () {
 	}
 
 	/**
+	 * Display a site-wide notice on missing settings for the widget.
+	 *
+	 * For users that have upgraded from v0.x to v1 the signup form will still
+	 * work but we want them to add an access token instead.
+	 */
+	public function displayNoticeInfo () {
+		echo '<br><div class="update-nag">' . sprintf(
+			__('MailMojo Widget is incomplete! <a href="%s">Please update settings now</a>.', 'mailmojo'),
```

Comment:
I suggest "Settings for the MailMojo widget is outdated. Please update the settings now."